src/ui/ResultList/ResultContainer.ts

```diff
@@ -19,7 +19,7 @@ export class ResultContainer {
 
   public empty() {
     this.searchInterface.detachComponentsInside(this.resultContainerElement.el);
-    this.resultContainerElement.el.innerHTML = '';
+    $$(this.resultContainerElement).empty();
```

Comment:
Is there a reason why we use the empty method?
If we simply want better performance, using the following code would be faster
`While (resultContainerElement.el.firstChild) {`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `resultContainerElement.el.removechildren(resultContainerElement.el.firstChild)`
`}`
Here's a benchmark i found about this: https://jsperf.com/innerhtml-vs-removechild/37

Reply:
That's a good suggestion, I checked the implementation of `empty` and it uses that logic. So we have the performance benefits encapsulated in an easy-to-use method :)

Reply:
Ok Great